scripts/variantstore/wdl/extract/vds_validation.py

```diff
@@ -42,18 +42,26 @@ def check_densify_small_region(vds):
 
 
 def main(vds):
-    check_samples_match(vds)
-    check_ref_blocks(vds)
     check_densify_small_region(vds)
-    vds.validate(); print('Hail VDS validation successful')
+    chrs = [f'chr{c}' for c in itertools.chain(range(1, 23), ['X', 'Y', 'M'])]
+    for chromosome_to_validate in chrs:
+        filtered_vd = vds.variant_data.filter_rows(vds.variant_data.locus.contig == chromosome_to_validate)
+        filtered_rd = vds.reference_data.filter_rows(vds.reference_data.locus.contig == chromosome_to_validate)
+        filtered_vds = hl.vds.VariantDataset(filtered_rd, filtered_vd)
+        print(f"Validating VDS chromosome {chromosome_to_validate}...")
+        check_samples_match(filtered_vds)
+        check_ref_blocks(filtered_vds)
+        filtered_vds.validate()
```

Comment:
Why are we validating all the filtered chromosome VDSes? There's a full VDS `validate()` on line 57

Reply:
Okay, I've removed that.